js/pro/wazirx.js

```diff
@@ -0,0 +1,607 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const wazirxRest = require ('../wazirx');
+const { NotSupported, ExchangeError } = require ('../base/errors');
+const { ArrayCacheBySymbolById } = require ('./base/Cache');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class wazirx extends wazirxRest {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'has': {
+                'ws': true,
+                'watchBalance': true,
+                'watchTicker': true,
+                'watchTickers': true,
+                'watchTrades': false,
+                'watchMyTrades': true,
+                'watchOrders': true,
+                'watchOrderBook': true,
+                'watchOHLCV': false,
```

Comment:
can't we implement `watchOHLCV` using this stream?
https://docs.wazirx.com/#kline-candlestick-stream